smtp_spam_detector/smtp_spam_detector_files/smtp_daemon.py

```diff
@@ -38,20 +38,20 @@
 
 """
 
-#!/usr/bin/env python
+#!/usr/bin/env python3
 import pytrap
 import sys
 import signal
 import logging
 import argparse
 from threading import Thread, Semaphore, Lock
-from flow import Flow, SMTP_Flow, FlowConversionException
-from smtp_entity import SMTP_ENTITY
-from detection import SpamDetection
+from .flow import Flow, SMTP_Flow, FlowConversionException
+from .smtp_entity import SMTP_ENTITY
+from .detection import SpamDetection
 try:
     from queue import Queue
 except:
-    from Queue import Queue
+    from queue import Queue
```

Comment:
This try-except block was used because of py2&py3 compatibility and it's useless now.